src/gwt/src/org/rstudio/studio/client/workbench/model/SessionInfo.java

```diff
@@ -56,6 +56,10 @@ public final native String getClientVersion() /*-{
    public final native String getUserIdentity() /*-{
       return this.userIdentity;
    }-*/;
+   
```

Comment:
Might be worth adding a note on this accessor as well as that's where I think most attempts to resolve paths on the client will land.